src/main/java/suppliers/ColorSupplier.java

```diff
@@ -0,0 +1,21 @@
+package suppliers;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    public static final String[] colours = {"white", "black", "red", "orange",
+            "yellow", "green", "blue", "purple"};
```

Comment:
what about constant naming? You can see examples [here](https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names)